addons/l10n_es/data/template/account.tax-es_common.csv

```diff
@@ -423,26 +423,26 @@
-"","","","","","","","","","","","","","","","","","","tax","","refund","account_common_473","","","",""
-"account_tax_template_s_irpf18","","True","18% Withholding income tax","Withholding 18%","sale","18% WHI","Retenciones a cuenta IRPF 18%","Retencions a compte IRPF 18%","-18.0","percent","tax_group_retenciones_18","","retencion","","","","","base","","invoice","","","Retención 18%","","tax_excluded"
-"","","","","","","","","","","","","","","","","","","tax","","invoice","account_common_473","","","",""
-"","","","","","","","","","","","","","","","","","","base","","refund","","","","",""
-"","","","","","","","","","","","","","","","","","","tax","","refund","account_common_473","","","",""
-"account_tax_template_s_irpf19","","True","19% Withholding income tax","Withholding 19%","sale","19% WHI","Retenciones a cuenta IRPF 19%","Retencions a compte IRPF 19%","-19.0","percent","tax_group_retenciones_19","","retencion","","","","","base","","invoice","","","Retención 19%","","tax_excluded"
-"","","","","","","","","","","","","","","","","","","tax","","invoice","account_common_473","","","",""
-"","","","","","","","","","","","","","","","","","","base","","refund","","","","",""
-"","","","","","","","","","","","","","","","","","","tax","","refund","account_common_473","","","",""
-"account_tax_template_s_irpf19a","","True","19% Withholding (leases)","Withholding 19% (Lease)","sale","19% WH lease","Retenciones a cuenta 19% (Arrendamientos)","Retencions a compte 19% (arrendaments)","-19.0","percent","tax_group_retenciones_19","","retencion","","","","","base","","invoice","","","Retención 19% (Arrend.)","","tax_excluded"
-"","","","","","","","","","","","","","","","","","","tax","","invoice","account_common_473","","","",""
-"","","","","","","","","","","","","","","","","","","base","","refund","","","","",""
-"","","","","","","","","","","","","","","","","","","tax","","refund","account_common_473","","","",""
-"account_tax_template_s_irpf195a","","True","19.5% Withholding (leases)","Withholding 19,5% (Lease)","sale","19.5% WH lease","Retenciones a cuenta 19,5% (Arrendamientos)","Retencions a compte 19,5% (arrendaments)","-19.5","percent","tax_group_retenciones_19-5","","retencion","","","","","base","","invoice","","","Retención 19,5% (Arrend.)","","tax_excluded"
-"","","","","","","","","","","","","","","","","","","tax","","invoice","account_common_473","","","",""
-"","","","","","","","","","","","","","","","","","","base","","refund","","","","",""
-"","","","","","","","","","","","","","","","","","","tax","","refund","account_common_473","","","",""
+"account_tax_template_s_irpf9","","True","9% Withholding income tax ","Withholding 9%","sale","9% WHI","Retenciones a cuenta IRPF 9%","Retencions a compte IRPF 9%","-9.0","percent","tax_group_retenciones_9","","retencion","","","","","base","+mod130[06]","invoice","","","Retención 9%","","tax_excluded"
```

Comment:
As it's Withholding taxes, I think it has to be `+mod130[06]` on first line. But otherwise I think you are right, I'm only using the tags for taxes and not invoices, so I'll delete the tags on invoice and keep the `+|-mod130[06]` on taxes.

Reply:
Did you forget to push?